cmd/trigger_ui/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<link rel="stylesheet" href="css/style.css">
```

Comment:
Did we consider importing Tailwind? 

I find it easier to work with and read declarative styling, and performance here isn't too important.